src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -52,12 +53,15 @@ internal partial class InteractiveWindow : IInteractiveWindow, IInteractiveWindo
             _intellisenseSessionStackMap = intellisenseSessionStackMap;
             _smartIndenterService = smartIndenterService;
 
-            var textContentType = contentTypeRegistry.GetContentType("text");
             var replContentType = contentTypeRegistry.GetContentType(PredefinedInteractiveContentTypes.InteractiveContentTypeName);
             var replOutputContentType = contentTypeRegistry.GetContentType(PredefinedInteractiveContentTypes.InteractiveOutputContentTypeName);
 
             _outputBuffer = bufferFactory.CreateTextBuffer(replOutputContentType);
             _standardInputBuffer = bufferFactory.CreateTextBuffer();
+            _promptBuffer = bufferFactory.CreateTextBuffer();
```

Comment:
The content type is Text in each case. We could use custom content types and use those to determine buffer kind rather than comparing against specific ITextBuffer instances. Comparing ITextBuffers was easier but custom content types may be cleaner.